src/config.ts

```diff
@@ -118,6 +118,7 @@ export const BVE_VOL_MAP = new Map<SYMBOL, number>([
 ]); // BVE vol keep alligned with app & contract vol
 
 export const SO_STATES: [string, PublicKey][] = [
+  ['GSOBONK_LOYALTY_10', new PublicKey('47JR8VXLc32X74ZFHcSFwkdZLWLdrU1f3WxDtgQUBhwj')],
```

Comment:
Not used, but this is supposed to be the state address

Reply:
Oh only name matters?

Reply:
for now. only because we assume the baseMint is BONK which lets us compute the state address.

Reply:
Will handle with the updates for MNGO then